Sources/Recycling/ListViews/UserAds/Cell/UserAdsListEmphasizedActionCell.swift

```diff
@@ -132,9 +146,13 @@ public class UserAdsListEmphasizedActionCell: UITableViewCell {
         contentView.addSubview(actionWrapper)
         actionWrapper.addSubview(actionTitleLabel)
         actionWrapper.addSubview(actionDescriptionLabel)
-        actionWrapper.addSubview(button)
+        actionWrapper.addSubview(actionButton)
+
+        actionWrapper.addSubview(gradientWrapper)
+        gradientWrapper.layer.addSublayer(gradientLayer)
+
         actionWrapperHideConstraint = NSLayoutConstraint(item: actionWrapper, attribute: .height, relatedBy: .equal, toItem: nil, attribute: .height, multiplier: 1, constant: 0)
-        actionWrapperHeightConstraint = NSLayoutConstraint(item: actionWrapper, attribute: .bottomMargin, relatedBy: .equal, toItem: button, attribute: .bottomMargin, multiplier: 1, constant: .largeSpacing)
+        actionWrapperHeightConstraint = NSLayoutConstraint(item: actionWrapper, attribute: .bottomMargin, relatedBy: .equal, toItem: actionButton, attribute: .bottomMargin, multiplier: 1, constant: .largeSpacing)
```

Comment:
Is there any difference between creating constraints like this versus for instance `actionWrapper.bottomAnchor.constraint(equalTo: actionButton.bottomAnchor, constant: .largeSpacing)`?

Reply:
No, not that I know of other than it being an older API.